src/mudlet-lua/lua/LuaGlobal.lua

```diff
@@ -146,25 +154,64 @@ if getMudletLuaDefaultPaths then
   end
 end
 
+if package.config:sub(1,1) == '\\' then
+  -- this is a windows platform, make all the path separators be '\\'
+  for i, path in ipairs(prefixes) do
+    prefixes[i] = string.gsub(path, '/', '\\')
+    if debugLoading then
+      echo("Directory separator conversion: \"" .. path .. "\" becomes: \"" .. prefixes[i] .. "\"\n")
```

Comment:
To avoid escaping all the double quotes, you may want to use single quotes or use `[[...]]` (double square braces) as string delimiters.

Reply:
I'll test it and see how the `[[`...`]]` work out.

Reply:
Have looked at it and have changed to use it...